Watchman.Discord/Areas/Protection/Services/MuteService.cs

```diff
@@ -112,37 +125,25 @@ private async Task AssignMuteRoleAsync(UserRole muteRole, UserContext userToMute
             Log.Information("User {user} has been muted on server {server}", userToMute.ToString(), server.Name);
         }
 
-        private MuteEvent GetNotUnmutedEvent(IEnumerable<MuteEvent> userMuteEvents)
-        {
-            return userMuteEvents.FirstOrDefault(x => x.Unmuted == false);
-        }
-
         private async Task UnmuteUser(UserContext mutedUser, MuteEvent muteEvent, DiscordServerContext serverContext)
         {
             var muteRole = this.GetMuteRole(serverContext);
-            await this.RemoveMuteRoleAsync(muteRole, mutedUser, serverContext);
+            await this._usersService.RemoveRole(muteRole, mutedUser, serverContext);
             await this.MarkAsUnmuted(muteEvent);
             Log.Information("User {user} has been unmuted on server {server}", mutedUser.ToString(), serverContext.Name);
         }
 
-        private async Task RemoveMuteRoleAsync(UserRole muteRole, UserContext userToUnmute, DiscordServerContext server)
-        {
-            await this._usersService.RemoveRole(muteRole, userToUnmute, server);
-        }
-
         private async Task MarkAsUnmuted(MuteEvent muteEvent)
         {
             var command = new MarkMuteEventAsUnmutedCommand(muteEvent.Id);
             await this._commandBus.ExecuteAsync(command);
         }
 
-        private IEnumerable<MuteEvent> GetMuteEvents(DiscordServerContext server, ulong userId)
+        private IEnumerable<MuteEvent> GetUserMuteEvents(DiscordServerContext server, ulong userId)
         {
             var getMuteEvents = new GetMuteEventsQuery(server.Id);
             var allServerMuteEvents = this._queryBus.Execute(getMuteEvents).MuteEvents;
-
-            var userMuteEvents = allServerMuteEvents.Where(x => x.UserId == userId);
-            return userMuteEvents;
+            return allServerMuteEvents.Where(x => x.UserId == userId);
```

Comment:
filtrowanie po user id powinno być na poziomie query handlera (jako opcjonalne)
żebyśmy w przyszłości mogli łatwo pobierać te informacje

przykładowo gdybyśmy chcieli wyświetlać liste zmutowanych użytkowników 
albo liste zmutowań konkretnego użytkownika

Reply:
do tego użyć PaginationQuery?

Reply:
możesz użyć
ale nie pamiętam czy tam jest opcja z userID

jak jest to fajnie
od razu do webowej sie przyda 

do webowej myśle że wszystko lub prawie wszystko powinno być jako PaginationQuery